config.sample.yaml

```diff
@@ -0,0 +1,92 @@
+---
+# After you have replaced all the __CHANGE_ME__ values in this file, you need to
+# run `make auth` to setup the Auth.
+management:
+  location: __CHANGE_ME__
+  mgmt_resource_group_name: __CHANGE_ME__
+  mgmt_storage_account_name: __CHANGE_ME__
+  terraform_state_container_name: tfstate
+  acr_name: __CHANGE_ME__
+  # Azure Resource Manager credentials used for CI/CD pipelines
+  arm_subscription_id: __CHANGE_ME__
+
+  # If you want to override the currently signed in credentials
+  # You would do this if running commands like `make terraform-install DIR=./templates/workspaces/base`
+  # arm_tenant_id: __CHANGE_ME__
+  # arm_client_id: __CHANGE_ME__
+  # arm_client_secret: __CHANGE_ME__
+
+tre:
```

Comment:
Offline I said I think we don't need this subsection and can move all children to be under the root.